### PR TITLE
2503: fixed. Image-style and manual crop added

### PR DESCRIPTION
**Pull request by KristianB14**
http://platform.dandigbib.org/issues/2503

### CODE REVIEW

File: modules/ding_frontend/ding_frontend.menu_block.inc [hunk 1]
```diff
@@ -28,21 +28,6 @@ function ding_frontend_menu_block_defaults() {
...
-  $menu_block->follow = '0';
-  $menu_block->level = 1;
-  $menu_block->parent = 'menu-secondary-menu:0';
-  $menu_block->sort = 0;
-  $menu_block->title_link = 0;
-  $export['2'] = $menu_block;
-
-  $menu_block = new stdClass();
-  $menu_block->disabled = FALSE; /* Edit this to true to make a default menu_block disabled initially */
-  $menu_block->api_version = 1;
```
Comment (kasperg) [inline modules/ding_frontend/ding_frontend.menu_block.inc]: Is it intentional that this menu is removed? It does not seem to be related to the change at hand.

File: modules/ding_frontend/ding_frontend.views_default.inc [hunk 2]
```diff
@@ -144,7 +144,6 @@ function ding_frontend_views_default_views() {
   $handler->display->display_options['arguments']['keys']['table'] = 'search_index';
   $handler->display->display_options['arguments']['keys']['field'] = 'keys';
   $handler->display->display_options['arguments']['keys']['default_action'] = 'empty';
-  $handler->display->display_options['arguments']['keys']['exception']['title'] = 'All';
```
Comment (kasperg) [inline modules/ding_frontend/ding_frontend.views_default.inc]: Same as above.

File: modules/ding_groups/ding_groups.strongarm.inc [hunk 3]
```diff
@@ -101,6 +101,15 @@ function ding_groups_strongarm() {
...
         ),
+        'metatags' => array(
+          'weight' => '40',
+        ),
+        'og_menu' => array(
+          'weight' => '0',
+        ),
+        'redirect' => array(
+          'weight' => '30',
+        ),
```
Comment (kasperg) [inline modules/ding_groups/ding_groups.strongarm.inc]: As above.

File: modules/ding_staff/ding_staff.views_default.inc [hunk 4]
```diff
@@ -730,27 +714,6 @@ function ding_staff_views_default_views() {
...
-      'override_label' => 0,
-      'label' => '',
-    ),
-    'action::pathauto_user_update_action' => array(
-      'selected' => 0,
-      'postpone_processing' => 0,
-      'skip_confirmation' => 0,
-      'override_label' => 0,
-      'label' => '',
-    ),
```
Comment (kasperg) [inline modules/ding_staff/ding_staff.views_default.inc]: As above.

**Comment by KristianB14** [comment]
The image-style crop_16_9 was missing. Cropping for events and staff is depending on this image-style, which is why they were not working. Now that the image-style is included, cropping for events and staff is working.

The changes unrelated to the issue have been reversed.
